@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -140,6 +140,11 @@ export interface SolidLineMaterialParameters extends FadingFeatureParameters {
      */
     depthTest?: boolean;
 
+    /**
+     * Enables/Disable depth write.
+     */
+    depthWrite?: boolean;
```

Comment:
Why do we need to expose depthWrite? Does not hurt but it is not needed to fix the issue.